slam-libraries/viam-cartographer/src/utils/slam_service_helpers.h

```diff
@@ -22,6 +52,26 @@ ActionMode DetermineActionMode(std::string path_to_map,
 // located in path_to_map.
 std::string GetLatestMapFilename(std::string path_to_map);
 
+/*
+casts the float f to a pointer of unsigned 8 bit bytes
+iterates throught all the 8 bit bytes of f
+writes each 8 bit bytes to buffer
+*/
+void writeFloatToBufferInBytes(std::string& buffer, float f);
```

Comment:
[nit/opt] writeIntToBufferInBytes -> writeIntToBufferInBinaryBytes or even included LittleEndian in the name?

Reply:
@nicksanford as well for your thoughts

Reply:
I do not think we need to tweak the name here but will differ if others believe we should

Reply:
I'm ok with tweaking the name at a later date given how long this PR has been open

Reply:
I think adding `Binary` to the name is unnecessary. Bytes are always in binary. Also, while I agree we should be more precise about endianness, especially given that the binary PCD format isn't specific about the expected endianness, is important, the way the code is written right now, will encode data in whatever endianness the processor supports, so specifying endianness in the function signature, would communicate intent, but may be misleading given that it will return big endian data if executed on a big-endian cpu.

I think more work needs to be done with regard to endianness guarantees for binary pcds, before changing the function names to imply endian guarantees we don't yet have.